quinn-proto/src/tests.rs

```diff
@@ -88,6 +88,7 @@ fn server_config() -> Config {
     };
 
     let mut tls_server_config = crypto::build_server_config();
+    tls_server_config.set_protocols(&[String::from_utf8(ALPN_QUIC_HTTP.to_vec()).unwrap()]);
```

Comment:
This would be cleaner as `str::from_utf8(ALPN_QUIC_HTTP).unwrap().into()` rather than bouncing through `Vec`